features/index.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+   As a User|
+   In order to navigate index.erb
+   I want to see a menu
+
+   Backgroud:
```

Comment:
This is spelled `Background`. I suspect that step is not being used in your program...